hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsService.java

```diff
@@ -218,6 +218,15 @@ void resolveAlertsForTrigger(String tenantId, String triggerId, String resolvedB
     void sendData(Collection<Data> data) throws Exception;
 
     /**
+     * Send data into the alerting system for evaluation.
+     *
+     * @param data Not Null.  The data to be evaluated by the alerting engine.
+     * @param ignoreFiltering  An optimization. Set true *only* if you are sure the data is useful for evaluation.
+     * @throws Exception any problem.
+     */
+    void sendData(Collection<Data> data, boolean ignoreFiltering) throws Exception;
```

Comment:
OK, I think it's OK for now. I mean, it's the java API and we only have one external client, and that is our tight integration with metrics.  Going forward we can certainly iterate.  I didn't include any option via the rest-api.